packages/authentication/src/__tests__/unit/providers/authentication.provider.unit.ts

```diff
@@ -108,10 +108,10 @@ describe('AuthenticateActionProvider', () => {
     function givenAuthenticateActionProvider() {
       strategy = new MockStrategy();
       strategy.setMockUser(mockUser);
-      provider = new AuthenticateActionProvider(
-        () => Promise.resolve(strategy),
-        u => (currentUser = u),
-      );
+      // provider = new AuthenticateActionProvider(
```

Comment:
Needed to comment out code here. @jannyHou's PR, https://github.com/strongloop/loopback-next/pull/2746, needs to rewrite this test file.